dotCMS/src/main/java/com/dotcms/ai/util/OpenAIRequest.java

```diff
@@ -54,8 +54,7 @@ public static void doRequest(final String urlIn,
                                  final OutputStream out) {
 
         if (!appConfig.isEnabled()) {
-            Logger.debug(OpenAIRequest.class, "OpenAI is not enabled and will not send request.");
```

Comment:
why removing the logs, logs are good

Reply:
Yeah let's leave these

Reply:
You can call the `AppConfig.debugLogger(final Class<?> clazz, final Supplier<String> message)` which uses the  `com.dotcms.ai.debug.logging` prop to enable/disabled debugging dynamically.